lists2safebrowsing.py

```diff
@@ -364,7 +363,7 @@ def process_entity_whitelist(incoming, chunk, output_file,
 
     output_file.flush()
     output_size = os.fstat(output_file.fileno()).st_size
-    print("Entity whitelist(%s): publishing %d items; file size %d" % (
+    print("Entity list(%s): publishing %d items; file size %d" % (
         list_variant, publishing, output_size))
```

Comment:
Although flake8 doesn't complain, I'd suggest fixing the indentation here a little

```suggestion
    print("Entity list(%s): publishing %d items; file size %d" 
          % (list_variant, publishing, output_size))
```